recipes/libnettle/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "nettle" %}
+{% set version = "3.3" %}
+{% set sha256 = "46942627d5d0ca11720fec18d81fc38f7ef837ea4197c1f630e71ce0d470b11e" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]  
+
+requirements:
+  build:
+    - m4
+    - gmp 6.1.*
```

Comment:
So I think we still need `gmp` in `run` as well, right?

Reply:
Apparently not with --enable-mini-gmp. I'm refering the error below if I don't use --enable-mini-gmp

Reply:
Ah, so is this a vendored, slimmed down copy of `gmp`? If so, maybe we don't need `gmp` here at all.

Though as @isuruf notes it would be better to use `gmp` if we can. As he asked, it would be good to see the logs where `gmp` encounters issues.

In any event, would defer to @isuruf judgement on all of this.

Reply:
Indeed he finaly fixed it :) Thank you for your help too !